testsuite/oauth/authorize/base.py

```diff
@@ -1,10 +1,13 @@
 """ Authorize the SMART API.
 """
+import time
 from abc import ABCMeta
 from urllib import parse
+from pyvirtualdisplay import Display
```

Comment:
Nitpick, but: I generally try to alphabetize the import statements (so `time` would go last) and they are also organized into groups: stdlib, pip modules, and local modules. So pyvirtualdisplay would go down with selenium.

https://www.python.org/dev/peps/pep-0008/#imports